spark/src/test/scala/geotrellis/store/GeoTrellisRasterSourceSpec.scala

```diff
@@ -95,11 +95,14 @@ class GeoTrellisRasterSourceSpec extends FunSpec with RasterMatchers with GivenW
         GeoTiffReader
           .readMultiband(TestCatalog.filePath, streaming = false)
           .raster
-          .resample((sourceMultiband.cols * 0.95).toInt, (sourceMultiband.rows * 0.95).toInt, NearestNeighbor)
-          // resample to 0.9 so RasterSource picks the base layer and not an overview
+          .resample(sourceMultiband.cols.toInt, sourceMultiband.rows.toInt, NearestNeighbor)
 
-      val resampledSource =
-        sourceMultiband.resample(expected.tile.cols, expected.tile.rows, NearestNeighbor)
+      val resampledSource = sourceMultiband.resample(
+        expected.tile.cols,
+        expected.tile.rows,
+        NearestNeighbor,
+        strategy = Base
```

Comment:
It was. When Nathan and I dug into this, we'd always get a raster at the base cell size. So it seems more correct to remove the fudge factor that attempts to get a non-base resolution and just have the resample operation pick the Base overview explicitly. 

Similar issue for the reproject test. When we load a raster with the GeoTiff method and then use reproject instead of resample method, that doesn't allow us to select an overview strategy since reproject doesn't care about that fundamentally. So again, we just use Base where necessary to preserve the idea that when we load the expected raster we're always dealing with the base resolution.

Hopefully that makes sense, we can discuss offline if more detail is necessary.

Reply:
I did revert to using AutoHigher in the RasterSourceRDD specs, since in those cases we load both in a way that we're resampling anyways.

Reply:
@CloudNiner i would like to revert all the strategies to AutoHigher; to preserve the old tests. This strategy is pretty important for RF and if we need more tests, lest create new tests without killing the old ones; how do you feel about that?

Reply:
I mean just as a check that the behavior of old functions did not change; if we’ll change tests, it is hard to say, was the old behavior preserved or not.

Reply:
Then theres a chunk of extra work here to get this build green, as I'm not sure how to get GeoTiffReader.readMultiband to be explicit about not choosing the Base cell size for its operations. It might be best if someone else takes that work on at this point.

Reply:
@CloudNiner how is that possible that old operations can't work as they worked before with AutoHigher? What has changed?

Reply:
After a discussion and the adjustments made that were discussed in https://github.com/locationtech/geotrellis/pull/3200#discussion_r401672704, @pomadchin is going to dig deeper as to why a number of tests are still failing.